modules/mtc-vm-creating-mig-plan.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="mtc-vm-creating-mig-plan_{context}"]
+= Creating the migration plan
+
+To migrate the `mig-vm` namespace, ensure that the `namespaces` field of the migration plan includes `mig-vm`.
+
+Modify the contents of the migration plan by adding `mig-vm` to the namespaces.
+
+.Example YAML
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigPlan
+metadata:
+  name: live-migrate-plan
+  namespace: openshift-migration
+spec:
+  namespaces:
+  - mig-vm # <1>
+...
+----
+<1> add `mig-vm` to the namespaces.
+
+In order to attempt a storage live migration the `liveMigrate` field in the spec must be set to true (and KubeVirt must be configred and be enabled to perform storage live migration.
+
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigPlan
+metadata:
+  name: live-migrate-plan
+  namespace: openshift-migration
+spec:
+  liveMigrate: true
+  namespaces:
+[... snipped, see config/samples/mig-plan.yaml for other required fields ...]
+----
+
+Live migration only happens during the cutover of a migration plan. Staging the migration plan skips any running virtual machines and does not sync the data. Any stopped virtual machine disks will be synced.
+
+During the migration, a `MigMigration` resource is created indicating what type of migration is happening:
+
+* stage
+* rollback
+* cutover
+
```

Comment:
Lines 51 forward: Maybe present simple instead of future?

Reply:
Do we have more detailed explanation for what is stage/rollback/cutover?

Reply:
> Do we have more detailed explanation for what is stage/rollback/cutover?

@duyanyan , this from _[Table 1. MTC terminology](https://docs.openshift.com/container-platform/4.16/migration_toolkit_for_containers/about-mtc.html#migration-terminology_about-mtc)_?
![image](https://github.com/user-attachments/assets/db13deb5-2693-4170-9e78-36c60568dd4b)

Reply:
@rayfordj - add an xref to this table? WDYT?

Reply:
> @rayfordj - add an xref to this table? WDYT?

@anarnold97 , I'm inclined to say, "_yes_", but this is an area of docs where I'm ignorant.  Referencing the existing table makes sense and I think if using 'xref' is the way we do that here then, "yes, add an xref to this table" :sweat_smile: